src/problems/urls.py

```diff
@@ -0,0 +1,11 @@
+from django.conf.urls import url
+
+from . import views
+
+app_name = 'problems'
+urlpatterns = [
+    # ex: /problems/
+    url(r'^$', views.index, name='index'),
+    # ex: /problems/5/
+    url(r'^(?P<problem_id>[0-9]+)/$', views.detail, name='detail'),
```

Comment:
Can we do URL's like /problems/finding_earthquake_epicenters/ or is it a little annoying to change around for now?